pax-exam-container-carbon/org.wso2.carbon.container/src/main/java/org/wso2/carbon/container/options/CarbonDistributionOption.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.container.options;
+
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.options.MavenArtifactUrlReference;
+import org.ops4j.pax.exam.options.MavenUrlReference;
+
+import java.nio.file.Path;
+
+/**
+ * Utility class to provide an easy and intuitive way to configure the specific carbon distribution options.
+ */
+public class CarbonDistributionOption {
+
+    /**
+     * Hidden utility class constructor.
+     */
+    private CarbonDistributionOption() {
+    }
+
+    /**
+     * Set the carbon distribution maven url option.
+     *
+     * @return an option to set the path to distribution.
+     */
+    public static CarbonDistributionBaseOption carbonDistribution(MavenUrlReference mavenUrlReference) {
+        return new CarbonDistributionBaseOption().distributionMavenURL(mavenUrlReference);
+    }
+
+    /**
+     * Set the carbon distribution path option.
+     *
+     * @return an option to set the path to distribution.
+     */
+    public static CarbonDistributionBaseOption carbonDistribution(Path path) {
```

Comment:
In pax-exam, for options classes, they have followed this approach. Check for all the options classes here[1].

[1] https://github.com/ops4j/org.ops4j.pax.exam2/tree/master/core/pax-exam/src/main/java/org/ops4j/pax/exam/options